spec/spec_helper.rb

```diff
@@ -11,6 +11,6 @@
 
   config.expect_with :rspec do |c|
     c.syntax = :expect
-    c.max_formatted_output_length = nil
+    # c.max_formatted_output_length = nil
```

Comment:
Do we want to remove this line?

Reply:
Yes, I was going back and forth on that during testing. I'll pull it out.